mplaltair/_convert.py

```diff
@@ -1 +1,104 @@
+# mapping of altair channel to mpl kwargs for scatter()
+_mpl_scatter_equivalent = {
+    'x': (lambda d: _process_x(d)),
```

Comment:
You do not need the lambda calls here, you can simply put the `_process_x` object into the dictionary.

In python functinos are first-class objects and can be passed around an such.